index.html

```diff
@@ -3,7 +3,57 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Portfolio project 1</title>
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+      rel="stylesheet"
+    />
+    <link href="./css/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+  <body class="body">
+    <nav class="first-color">
+      <h1 class="owner">TENIFAYO</h1>
+      <span class="click">&#9776;</span>
+    </nav>
+
+    <section class="about-me">
+      <h1 class="color margin font1">
+        Hey There . <br />
+        I`m Tenifayo
+      </h1>
+      <p class="margin font2">I`m a Software Developer</p>
+      <p class="margin font3">
+        I can help you build a product , feature or website look through some of my work and
+        experience! if you like what you see and have a project you need coded, don`t hesitate to
+        contact me
+      </p>
+
+      <ul class="edit-icon">
+        <li class="spce-icon">
+          <a href="https://github.com/abdulsomod1" title="github"
+            ><i class="fa-brands fa-github edit-link"></i
+          ></a>
+        </li>
+        <li class="spce-icon">
+          <a href="https://www.linkedin.com/in/abdulsomod1/" title="linkedin"
+            ><i class="fa-brands fa-linkedin-in edit-link"></i
+          ></a>
+        </li>
+        <li class="spce-icon">
+          <a
+            href="https://venture.angellist.com/v/abdulsomod-tenifayo/messages/new"
+            title="angellist"
+            ><i class="fa-brands fa-angellist edit-link"></i
+          ></a>
+        </li>
+        <li class="spce-icon">
+          <a href="https://twitter.com/CodingC31597" title="twitter"
+            ><i class="fa-brands fa-twitter edit-link"></i
+          ></a>
+        </li>
+      </ul>
+    </section>
+
+    <script src="https://kit.fontawesome.com/a076d05399.js" crossorigin="anonymous"></script>
```

Comment:
Let's remove fontawesome, it will only add unnecessary file size that will result in a slower page load, instead, let's import the svg from the Figma design and use them directly.